Twig/DoctrineExtension.php

```diff
@@ -327,7 +328,12 @@ public function highlightSqlSyntax($sql)
         \SqlFormatter::$comment_attributes = 'class="comment"';
         \SqlFormatter::$variable_attributes = 'class="variable"';
 
-        $html = \SqlFormatter::highlight($sql);
+        if ($highlightOnly) {
+            $html = \SqlFormatter::highlight($sql);
+        } else {
+            $html = \SqlFormatter::format($sql);
+        }
+
         $html = preg_replace('/<pre class="(.*)">([^"]*+)<\/pre>/Us', '<div class="\1"><pre>\2</pre></div>', $html);
```

Comment:
We can't use `$use_pre` because is a "global state" and we need to output different things depending whther we highlight or not. I've just updated the code.